src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -197,6 +201,14 @@ class ProfileCallTreeContextMenu extends PureComponent {
         <MenuItem onClick={this.handleClick} data={{ type: 'focus-subtree' }}>
           Focus on subtree
         </MenuItem>
+        {inverted
```

Comment:
I think you could have this menu item visible in all modes. Xcode calls this "Focus on calls made by <function>", so it's clear in which direction you're going regardless of the current view mode.